assets/components/cards/render/WireListCard.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import MoreNewsButton from './MoreNewsButton';
+import classNames from 'classnames';
+import {characterCount, wordCount} from 'utils';
+import WireListItemIcons from 'wire/components/WireListItemIcons';
+import {FieldComponents} from 'wire/components/fields';
+import {Embargo} from 'wire/components/fields/Embargo';
+import {UrgencyItemBorder, UrgencyLabel} from 'wire/components/fields/UrgencyLabel';
+import {isKilled, getCaption, getImageForList} from 'wire/utils';
+import {connect} from 'react-redux';
+import {listConfigSelector} from 'ui/selectors';
+import {DEFAULT_COMPACT_META_FIELDS} from 'wire/components/WireListItem';
+import {IArticle, IListConfig} from 'interfaces';
+import {showModal} from '@superdesk/common';
+import {filterGroupsToLabelMap} from 'search/selectors';
+
+interface IOwnProps {
+    item: IArticle;
+}
+
+interface IPropsReduxStore {
+    listConfig: IListConfig;
+    filterGroupLabels: any;
+}
+
+type IPropsCombined = IPropsReduxStore & IOwnProps;
+
+class WireListPanel extends React.Component<IPropsCombined> {
+    wordCount: number;
+    characterCount: number;
+
+    constructor(props: IPropsCombined) {
+        super(props);
+
+        this.wordCount = wordCount(props.item);
+        this.characterCount = characterCount(props.item);
+    }
+
+    render(): React.ReactNode {
+        const cardClassName = classNames('wire-articles__item-wrap col-12 wire-item');
+        const wrapClassName = classNames('wire-articles__item wire-articles__item--wire wire-articles__item--list');
+        const listImage = getImageForList(this.props.item);
+        const compactFields = this.props.listConfig.compact_metadata_fields || DEFAULT_COMPACT_META_FIELDS;
+        const {item, listConfig} = this.props;
+        const isExtended = false;
+
+        // FIXME: Implement the proper subscription overlay
+        const showOverlay = () => showModal(({closeModal}) => {
```

Comment:
This dashboard card should behave in the same way that all other cards behave, i.e. when clicking it should open the wire item in full page preview.

At this present stage, none of the dashboard cards should have any code for the public dashboard feature we will be building.

Reply:
Ok will change this